### PR TITLE
Improvements to 2SV reset for user/administrator

### PR DESCRIPTION
**Pull request by benlovell**
Trello: https://trello.com/c/diHw4iMo

### Unexpanded
screen shot 2015-10-27 at 08 09 23

### Expanded
screen shot 2015-10-27 at 08 09 38

### Administrative text
screen shot 2015-10-27 at 08 16 16

### CODE REVIEW

**Comment by boffbowsh** [comment]
I hate to be "that guy", but the nesting div is a couple of pixels too far to the right. Maybe na issue with `govuk_admin_template` @fofr?

**Comment by benlovell** [comment]
@boffbowsh 

b64wfyuciaajq3u

**Comment by benlovell** [comment]
@fofr I rebased the smart quotes in. I even added them to the toggle link. :star2:

**Comment by benlovell** [comment]
@fofr I also rebased in a smaller (`h4`) heading.

screen shot 2015-10-27 at 10 50 43

File: app/views/devise/two_step_verification_session/new.html.erb [hunk 1]
```diff
@@ -23,6 +23,15 @@
           autocomplete: 'off',
           'aria-describedby': 'code-hint' %>
       </div>
+      <div data-module="toggle" class="if-no-js-hide">
+        <%= link_to "Can’t get your verification code?", "#", class: "js-toggle js-toggle-target" %>
+        <div class="js-toggle-target if-js-hide">
+          <h4>Contact <%= link_to "GOV.UK", "https://www.gov.uk/" %></h4>
```
Comment (fofr) [inline app/views/devise/two_step_verification_session/new.html.erb]: I'm not sure this should link, I think that's not a helpful link in this context.

File: app/views/devise/two_step_verification_session/new.html.erb [hunk 2]
```diff
@@ -23,6 +23,15 @@
           autocomplete: 'off',
           'aria-describedby': 'code-hint' %>
       </div>
+      <div data-module="toggle" class="if-no-js-hide">
```
Comment (fofr) [inline app/views/devise/two_step_verification_session/new.html.erb]: Almost missed this one. The `if-no-js-hide` should be on the link below. If there's no JS we want to show the text without the toggle.

Comment (benlovell) [inline app/views/devise/two_step_verification_session/new.html.erb]: :dizzy_face: whoops, good catch.

**Pull request merged**